### DISCUSSION TITLE
Different IPDF representation depending on number of points

### DISCUSSION THREAD

**Discussion by dalbinsson** [mtex-toolbox/mtex#911] [Ask Anything]
Hi, 

When plotting a set of orientations in an IPDF I get different points represented depending on the number of points i plot.
I am interested in grains with a Palladium crystal structure (m-3m) and I want to  plot the IPDF in the "Z" direction. So I here generate a set of 100 random orientations with the corresponding crystal  symmetry. If i plot fewer than 100 particles, i see a difference between the +Z and -Z direction, where some particles are shown only in the +Z direction and others only in the -Z direction. But if a plot 100 or more, both figures are identical. I have attached two example figures to illustrate the results.
What might cause this problem? 

The command used for all plots is plotIPDF(orientations(1:n),[vector3d.Z -vector3d.Z])

[IPDF_99orientations.pdf](https://github.com/mtex-toolbox/mtex/files/6211433/IPDF_99orientations.pdf)
[IPDF_100orientations.pdf](https://github.com/mtex-toolbox/mtex/files/6211443/IPDF_100orientations.pdf)

**Comment by ralfHielscher** [comment]
Hi,

I can not reproduce this issue. Could you please report which version of MTEX you are using. Have you tested it with the latest version of MTEX?

Ralf.

**Comment by dalbinsson** [comment]
Hi Ralf,

Thanks for this fast reply.
I had version 5.5 from December and now downloaded the newest (5.6..) after
this  all points are shown for both 99 and 100 orientations so the problem
seems to be fixed.

Best regards,
David

Den fre 26 mars 2021 kl 12:49 skrev Ralf Hielscher ***@***.***
